test/build.bats

```diff
@@ -137,6 +137,31 @@ load common
     rm -Rf --one-file-system "$tar" "$img"
 }
 
+@test 'ch-pull2tar' {
+    scope standard
+    need_docker
+    tag='alpine:latest'
+    tar="${ch_tardir}/${tag}.tar.gz"
```

Comment:
Might use `$CH_TEST_TMPDIR` rather than `$ch_tardir`.

Reply:
I do think it is a bit awkward to unpack to our tarball directory instead of a temp directory. However, since we do this for both `ch-pull2dir` and `ch-build2dir`, we should probably make a new issue to address both if we decide that is the approach we want to take.

Reply:
Yes, let's solve both separately.